### PR TITLE
Don't shorten long API names as customers depend on them

### PR DESCRIPTION
**Pull request by thomasmaas**
closes https://issues.jboss.org/browse/THREESCALE-1550

### CODE REVIEW

File: app/assets/stylesheets/provider/admin/_dashboard.scss [hunk 1]
```diff
@@ -29,12 +29,9 @@
     font-weight: $font-weight-bold;
     color: $font-color;
     float: left;
-    max-width: line-height-times(10);
     margin: 0 0 0 0;
```
Comment (mayorova) [inline app/assets/stylesheets/provider/admin/_dashboard.scss]: Don't you want to add some right margin to make sure Overview doesn't get too close to the service name?

Comment (damianpm) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: Agree, if no right-margin this happens:
no-margin
We can do `margin: 0 2rem 0 0`

Comment (thomasmaas) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: it can't, it would wrap to next line.

Comment (damianpm) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: It depends on the size of the screen, If you resize at some point it could happen

Comment (damianpm) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: It depends on the size of the screen, if you resize at some point it could happen

Comment (thomasmaas) [reply app/assets/stylesheets/provider/admin/_dashboard.scss]: Thanks, you are both right. I need coffee…

**Comment by thomasmaas** [comment]
Thank you @damianpm & @mayorova

**Pull request merged**